src/rpc/core_rpc_server.h

```diff
@@ -202,9 +205,17 @@ namespace cryptonote
     //utils
     uint64_t get_block_reward(const block& blk);
     bool fill_block_header_response(const block& blk, bool orphan_status, uint64_t height, const crypto::hash& hash, block_header_response& response);
+    enum invoke_http_mode { JON, BIN, JON_RPC };
```

Comment:
If it's your own type, why JON and not JSON ?

Reply:
Because I'm assuming that JON as used in https://github.com/monero-project/monero/blob/master/src/rpc/core_rpc_server.h isn't a typo. If it's indeed a typo, I'll happily use JSON:)

Reply:
Since other stuff uses JON, it's fine to use this too as it makes clear it relates to that.